test/e2e/density.go

```diff
@@ -155,6 +155,12 @@ var _ = Describe("Density", func() {
 			// Tune the threshold for allowed failures.
 			badEvents := BadEvents(events)
 			Expect(badEvents).NotTo(BeNumerically(">", int(math.Floor(0.01*float64(totalPods)))))
+
+			// Verify latency metrics
+			// TODO: update threshold to 1s once we reach this goal
+			highLatencyRequests, err := HighLatencyRequests(c, 10*time.Second)
```

Comment:
Reset this before the test? otherwise could pass or fail depending on what you ran before running this test, no? (e.g. if you just ran the rest of the e2e suite, then there'd be enough extra OK requests that the offending requests are pushed into the 99th %ile.)